converters/code.go

```diff
@@ -0,0 +1,29 @@
+package converters
+
+import "strings"
+
+// FormatJavaCodeStr will remove from the provided java code new lines and white space's
```

Comment:
`// FormatPainlessSource will clean all new lines and white spaces from the input`

Reply:
done.